### DISCUSSION TITLE
Please consider forking `gnome-screenshot`, the screenshot tool Cinnamon currently uses, because the GNOME developers may stop updating that package very soon.

### DISCUSSION THREAD

**Discussion by dazzletune** [linuxmint/discussions#524] [Development] [answered]
`gnome-screenshot` is a screenshot tool used by Cinnamon and formerly GNOME prior to GNOME 42. However, since GNOME 42, GNOME Shell now uses a screenshot and screen recorder that is directly integrated into the desktop environment rather than being a separate package. Given that the GNOME developers will likely stop updating the `gnome-screenshot` package, and the Linux Mint team's past decisions to fork GNOME applications like Gedit, Nautilus, GNOME-Control-Center, and Evince, the Linux Mint team should consider forking `gnome-screenshot` into `cinnamon-screenshot`.

**Comment by goyalyashpal** [comment]
[xfce]: https://search.nixos.org/packages?show=xfce.xfce4-screenshooter&query=xfce%20screenshot
[mate]: https://search.nixos.org/packages?show=mate.mate-utils&query=mate-screenshot

bumping this.
proposed existing alternate:

- Xfce's [screenshooter][xfce] tool supports all these formats (see [here](https://9to5linux.com/xfces-screenshot-tool-now-lets-you-save-screenshots-in-avif-and-jxl-image-formats)).

    Support additional file types like WebP, AVIF, JPEG XL
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/234


- mate [screenshot][mate] by @ht-7 in #\\219

---
[here]: https://github.com/orgs/linuxmint/discussions?discussions_q=screenshot&page=2

requests [here]:
* #492 
* #347 
* #219 




---
[updates]: https://gitlab.gnome.org/GNOME/gnome-screenshot/-/commits/master
[Alice Mikhaylenko]: https://gitlab.gnome.org/alicem

Not maintained notes [upstream]:


> gnome-screenshot is being retired anyway, screenshots are done by the shell itself now and it can do whatever it wants.
>
> \- _[Alice Mikhaylenko] · [2022-Mar](https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/190#note_1403713) · `#190` annotation feature_



> Since we have a new UI in 42 and gnome-screenshot isn't in core anymore, I'm not maintaining it anymore. :)
>
> \- _Remove myself from doap • cd3626b1_
> _[Alice Mikhaylenko] authored [2022-March](https://gitlab.gnome.org/GNOME/gnome-screenshot/-/commits/master?search=doap)_

---
[updates] history:
* rest all: translation updates
* minute updates in 2021-Nov,Sep,Apr,Mar
* minor updates: 2020-Sep,Oct,Aug
* major updates: 2020-Apr,Mar

**Reply by goyalyashpal** [reply]
----
feature requests pending upstream:

- Support additional file types like WebP, AVIF, JPEG XL
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/234

- In Wayland + Fractional Scaling enabled, capturing a window screenshot results in a very high resolution image
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/245

- When combining -p, -a and -d options no cursor is present on the screenshots
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/244

- "Take screenshot of Selected Area" from rigtclick of launcher
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/220

- Ability to select multiple regions
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/218

- New capture button on the post-capture dialogue.
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/204

- Remember Capture Area option
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/202

- Option to always automatically copy to clipboard.
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/203

- annotation feature
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/190

- move screen selection during selection
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/67

- Screenshots using "Select area to grab" have translucent overlay
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/56

- take a scrolling screenshot
https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/54

**Comment by mtwebster** [comment, marked as answer]
https://github.com/linuxmint/cinnamon/commit/b780321697ddc1c13755743d7aa6ac8e1561c549
